wobgob/azerothcore
# Mod war effort

## Changes Proposed:
-  add in module mod-war-effort

## Tests Performed:

- logged in as both alliance and horde characters
- confirm all NPC are in Orgrimmar and Ironforge
- Hand in quest item
- 

## Known Issues and TODO List:


- NPC's visible when the module no set to be enabled
- Quests and Hand in's can be done with the module not enabled. but progress will not be saved to the DB.
